modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -654,4 +654,14 @@ public static class Marshalling {
         /** Unmarshalling error. */
         public static final int UNMARSHALLING_ERR = MARSHALLING_ERR_GROUP.registerErrorCode((short) 3);
     }
+
+    /** REST service error group. */
+    @ErrorCodeGroup
+    public static class Rest {
+        /** REST service error group. */
+        public static final ErrorGroup REST_ERR_GROUP = registerGroup("REST", (short) 23);
+
+        /** Cluster has not yet been initialized or the node is in the process of stopping. */
+        public static final int CLUSTER_NOT_INIT_ERR = REST_ERR_GROUP.registerErrorCode((short) 1);
```

Comment:
Why didn't you use already existing `org.apache.ignite.lang.ErrorGroups.Embedded#CLUSTER_NOT_INITIALIZED_ERR`?

Reply:
This is an error from group `Embedded`, as far as I can see this error can appear if you use Ignite (`org.apache.ignite.IgniteServer#initClusterAsync`) directly and call cluster initialization from the client code.
The error code I suggested will be from the REST service, which is a little different in my opinion.
What is your opinion?

Reply:
My opinion is mixed, but let's agree that your approach is right for now, and if anyone has any objection to is we'll fix it. Approve